packages/google-cloud/package.json

```diff
@@ -133,7 +115,8 @@
   },
   "scripts": {
     "publish-module": "node ../../scripts/publish.js google-cloud",
-    "test": "mocha test/*.js"
+    "test": "mocha test/*.js",
```

Comment:
Do we need to keep the test?

Reply:
I think I can delete the tests. I just realized if I keep them I need to also keep the dependencies. It likely makes sense to remove all of it.